### PR TITLE
capstone project - Budget app

### PR DESCRIPTION
**Pull request by looh16**
**For this project, functionalities were implemented to allow the user:**

1. Register and log in, so that the data is private to them.
2. Introduce new transactions associated with a category.
3. See the money spent on each category.

**Tests were implemented for:**

1. Views
2. Models

[Link Demo](https://www.loom.com/share/dc7cc6dda3fb44b48a57a7894c9ccca3)

### CODE REVIEW

**Comment by thecodechaser** [comment]
@looh16 Please check my message on slack

File: app/views/groups/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  </div>
+  
+  
+    <div class="field mt-3 d-flex flex-column">
+    <%= f.submit "Add New", class: "btn form-control btn-success green_button " %>
+  </div>
+    </div>
+
+  <% end %>
+</div>
```
Comment (thecodechaser) [inline app/views/groups/new.html.erb]: - [ ] Please make sure on `add new category` page, There's a button to go back to the homepage(category page) as requested in the project requirements.
   image


image

File: app/views/entities/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        </div>
+        </div>
+    <% end %>
+    </div>
+    <div class="container field mt-3 d-flex flex-column">
+    <%= link_to new_group_entity_path do %>
+    <button class="btn form-control btn-success green_button ">Add Transation</button>
+   <% end %>
+    </div>
+</div>
```
Comment (thecodechaser) [inline app/views/entities/index.html.erb]: - [ ] Please make sure on the `transaction` page, Users are able to see the total amount(Sum of all transaction) in that category as requested in the project requirements.
    image

image

**Comment by thecodechaser** [comment]
@looh16 Please check slack

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @looh16👋, 

### You are doing a marvelous job on the RoR capstone project and this is one of the best projects I have seen so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- All tests are passed. ✔️



## Required Changes ♻️

- [ ] Please make sure the project is deployed and accessible online as requested in the project requirements.

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by looh16** [comment]
heroku

I have an error after deploying, as shown in the image, that's why I couldn't insert the application link in the readme file, I'm trying to solve the error and then I'll insert it in the readme file

**Review by julie-ify — APPROVED** [review]
Hi @looh16,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by looh16** [comment]
Thank you!

**Pull request merged**